extensions/kind/package.json

```diff
@@ -81,7 +81,17 @@
           "icon": "${kind-icon}"
         }
       ]
-    }
+    },
+    "commands": [
+      {
+        "command": "kind.install",
+        "title": "Kind: Install..."
+      },
+      {
+        "command": "kind.image.move",
```

Comment:
This command is already declared in line 71 ?

Reply:
@jeffmaury the command is bound to a menu line 71 but it's still not declared